stackdriver/lib/legacy_stackdriver.rb

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2013 Grant T. Olson
+# All rights reserved.
+#
+#   Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are
+# met:
+#
+#   * Redistributions of source code must retain the above copyright
+# notice, this list of conditions and the following disclaimer.
+#
+#   * Redistributions in binary form must reproduce the above
+# copyright notice, this list of conditions and the following
+# disclaimer in the documentation and/or other materials provided
+#       with the distribution.
+#
+#     * Neither the name of the Grant T. Olson nor the names of
+#       additional contributors may be used to endorse or promote
+#       products derived from this software without specific prior
+#       written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+#
+# This file provides backward compatibility with legacy stackdriver v0.2.2 gem.
+# Original code can be found at https://github.com/sammarx/stackdriver-ruby
+#
+require 'multi_json'
+
+module StackDriver
+  POST_URI = "https://custom-gateway.stackdriver.com/v1/custom"
+  DELETE_URI = "https://custom-gateway.stackdriver.com/v1/delete_custom"
+
+  def self.init *args
+    # Deprecation message
+    puts "This API has been deprecated by Stackdriver. For new version of " \
```

Comment:
"This usage is specific to the legacy Stackdriver service. It is deprecated and will be removed at some point in the future. Please migrate to the Google Stackdriver API documented at ..."